client/packages/coldchain/src/Monitoring/api/TemperatureNotification/hooks/useTemperatureNotificationList.ts

```diff
@@ -22,7 +22,7 @@ export const useTemperatureNotificationList = (queryParams?: ListParams) => {
   const { temperatureNotificationApi, storeId } =
     useTemperatureNotificationGraphQL();
 
-  const queryKey = [TEMPERATURE_NOTIFICATION, storeId, LIST, queryParams];
+  const queryKey = [TEMPERATURE_NOTIFICATION, storeId, LIST_KEY, queryParams];
```

Comment:
Just fixing since I changed the key.... LIST was getting confusing with the list component XD